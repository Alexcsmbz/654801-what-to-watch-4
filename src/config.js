export const moviePageTabs = [
  {
    name: `Overview`,
  },
  {
    name: `Details`,
  },
  {
    name: `Reviews`,
  },
];

export const RoutePath = {
  MAIN: `/`,
  LOGIN: `/login`,
  MOVIE: `/films/:id`,
  MYLIST: `/mylist`,
  REVIEW: `/films/:id/review`,
};
