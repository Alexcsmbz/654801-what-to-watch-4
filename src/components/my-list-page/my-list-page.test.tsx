import MyListPage from './my-list-page';
import {BrowserRouter as Router} from 'react-router-dom';
import {testUser} from 'config';

describe(`MyListPage snapshot test`, () => {
  it(`MyListPage should render my movies list page`, () => {
    const tree = renderer.create(
        <Router>
          <MyListPage
            user={testUser}
            movies={[]}
            onClick={() => {}}
            getFavoriteMovies={() => {}}
          />
        </Router>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
