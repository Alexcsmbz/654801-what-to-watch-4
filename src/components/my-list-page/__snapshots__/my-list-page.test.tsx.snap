// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyListPage snapshot test MyListPage should render my movies list page 1`] = `
<div
  className="user-page"
>
  <header
    className="page-header user-page__head"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link undefined"
        href="/"
        onClick={[Function]}
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <h1
      className="page-title user-page__title"
    >
      My list
    </h1>
    <div
      className="user-block"
    >
      <div
        className="user-block__avatar"
      >
        <img
          alt="name avatar"
          height="63"
          src="https://4.react.pages.academyavatarUrl"
          width="63"
        />
      </div>
    </div>
  </header>
  <section
    className="catalog"
  >
    <h2
      className="catalog__title visually-hidden"
    >
      Catalog
    </h2>
    <div
      className="catalog__movies-list"
    >
      <div
        style={
          Object {
            "display": "grid",
            "justifyItems": "center",
            "margin": "auto",
          }
        }
      >
        <h2
          className="catalog__title"
        >
          You have no movies added yet ðŸŽ­
        </h2>
        <a
          className="catalog__title"
          href="/"
          onClick={[Function]}
        >
          Go home and add
        </a>
      </div>
    </div>
  </section>
  <footer
    className="page-footer"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link logo__link--light"
        href="/"
        onClick={[Function]}
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div
      className="copyright"
    >
      <p>
        Â© 2019 What to watch Ltd.
      </p>
    </div>
  </footer>
</div>
`;
